packages/cli/src/create-app/tests/copy-starter-template.test.ts

```diff
@@ -0,0 +1,56 @@
+import path from 'path';
+import fsExtra from 'fs-extra';
```

Comment:
```suggestion
import { copySync } from 'fs-extra';
```

Import syntax is wrong